.github/workflows/tag-and-build-release.yaml

```diff
@@ -0,0 +1,108 @@
+name: Tag and Build Release
+on:
+  workflow_dispatch:
+    inputs:
+      release_version:
+        description: new release version
+        required: true
+        default: (for example, 0.1.0)
+
+jobs:
+  checks:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check inputs
+        run: |
+          if [[ ! "${{ github.event.inputs.release_version }}" =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
```

Comment:
you don't want to allow SNAPSHOT or ...-BETA, ..-RC1 type version releases?

Reply:
we could, but I dunno, maybe it doesn't matter right now.